blocks/library/freeform/editor.scss

```diff
@@ -140,7 +140,7 @@ div[data-type="core/freeform"] .editor-block-contextual-toolbar + div {
 
 .freeform-toolbar {
 	width: auto;
-	margin: - $block-padding;
+	margin: -#{ $block-padding + 1 } -#{ $block-padding } -#{ $block-padding } -#{ $block-padding };
```

Comment:
I think this code deserves a comment. :)

Reply:
😂 

On it in a minute.

Reply:
I also lost the top border now.

<img width="718" alt="screen shot 2018-04-13 at 10 51 43" src="https://user-images.githubusercontent.com/4710635/38725786-b67d3a74-3f08-11e8-8899-f626af31c7f9.png">

Reply:
It works, computes px value :)